pki/constants.go

```diff
@@ -58,4 +58,8 @@ const (
 	KubeAdminOrganizationName = "system:masters"
 	KubeAdminConfigPrefix     = ".kube_config_"
 	KubeAdminConfigENVName    = "KUBECFG_ADMIN"
+
+	EtcdName              = "etcd"
```

Comment:
We should perhaps name it similar to KubeNodeCertENVName, to reflect the purpose